src/status_im/ui/components/bottom_bar/core.cljs

```diff
@@ -43,7 +43,6 @@
       {:nav-stack           :wallet-stack
        :content             {:title (i18n/label :t/wallet)
                              :icon  :main-icons/wallet}
-       :count-subscription  :get-wallet-unread-messages-number
```

Comment:
don't we want to use it for new transactions?

Reply:
We don't do anything with it yet, it's a subscription that returns 0. We can always add it back once it will be used